### ISSUE TITLE
Put summary entry after content in form, maybe make it optional

### ISSUE THREAD

**Issue by RolandStuder** [marcojakob/lilo-blog#6]
Currently I the summary field is presented above the content field, which I think is the wrong order. 

I rarely know exactly what I will write ahead of time, and to extract the gist of it in a summary is hard to do, if I haven't written the article out yet.

That's why I would move the summary field just after the content field, maybe also the picture field. Let me write first.

Also I would consider making the summary optional, as it is additional work, if it is missing you could just use the first XXX characters of the article itself.

**Comment by marcojakob** [comment]
I've moved the summary and image to the bottom. As you suggested we can use the first words of the content as the summary if nothing is provided.

With the standard template of the post where the image is also placed at the top of the post it feels a bit strange to move the image below the content. Do you agree @RolandStuder? I might move the image back up again.

**Comment by marcojakob** [comment]
BTW: Is this a problem that we're using the same image for the blog page, the blog overview and the meta image? It's currently not possible to provide a meta image and not show it in the blog.

**Comment by marcojakob** [comment]
This is the current solution:

I removed the featured image that was automatically added to the top of every blog post if an image was provided. This makes it possible to provide an image for the blog overview and SEO but not show it at the top of the blog post.

In the CMS I've made the image, tags, and summary `optional`. But placing them after the content seemed rather strange.

If no summary is provided, the first words of the content are used - like you suggested @RolandStuder.
